tools/packaging/kernel/build-kernel.sh

```diff
@@ -110,7 +110,7 @@ Options:
 	-t <hypervisor>	: Hypervisor_target.
 	-u <url>	: Kernel URL to be used to download the kernel tarball.
 	-v <version>	: Kernel version to use if kernel path not provided.
-	-x <type>	: Confidential guest protection type, such as sev, snp, tdx, or "confidential" (for all of those).
+	-x <type>	: All the confidential guest protection type for a specific architecture.
```

Comment:
Nit: This looks a bit odd as the only value currently is "confidential" so could we either remove the `<type>` arg here, or specify in the usage that the value must be "confidential"?

Reply:
I've removed the type, thanks for the suggestion!